src/ccmain/ltrresultiterator.cpp

```diff
@@ -61,7 +63,9 @@ char *LTRResultIterator::GetUTF8Text(PageIteratorLevel level) const {
       do {            // for each text line in a paragraph
         do {          // for each word in a text line
           best_choice = res_it.word()->best_choice;
-          ASSERT_HOST(best_choice != nullptr);
+          if (best_choice == nullptr) {
```

Comment:
Is there an example which triggers the assertion here? I am not sure whether the `break` is the right solution here. Maybe the loop should continue at line 71.